invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities_items/ComputerTabletCommunitiesItem.js

```diff
@@ -8,7 +8,8 @@ import { i18next } from "@translations/invenio_app_rdm/i18next";
 import _get from "lodash/get";
 import React from "react";
 import { Image } from "react-invenio-forms";
-import { Icon, Item, Label } from "semantic-ui-react";
+import { Icon, Item, Label, Grid } from "semantic-ui-react";
```

Comment:
The [line](https://github.com/inveniosoftware/invenio-communities/pull/835/files#diff-92f826f7079183b2697c7bbd983065844007d8d44ae983c59ee1bbe910ecd2e5R7) above is wrong! Can you change this?